src/MizarWidgets/SamplingWithFrameTrackWidgetTest.cpp

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+#include <qnamespace.h>
+
+#include <QApplication>
+#include <QCheckBox>
+#include <QLabel>
+#include <memory>
+
+#include "MizarWidgets/SamplingWithFrameTrackWidget.h"
+
+using ::testing::NotNull;
+
+namespace orbit_mizar_widgets {
+
+constexpr const char* kOrgName = "The Orbit Authors";
+
+constexpr std::string_view kMultiplicityCorrectionEnabledLabel =
+    "Probability of false-alarm for at least one function:";
+constexpr std::string_view kMultiplicityCorrectionDisabledLabel =
+    "Probability of false-alarm for an individual function:";
+
+class SamplingWithFrameTrackWidgetTest : public ::testing::Test {
+ public:
+  SamplingWithFrameTrackWidgetTest() : widget_(std::make_unique<SamplingWithFrameTrackWidget>()) {
+    QApplication::setOrganizationName(kOrgName);
+    QApplication::setApplicationName("SamplingWithFrameTrackInputWidgetTest.InitIsCorrect");
+
+    multiplicity_correction_ = widget_->findChild<QCheckBox*>("multiplicity_correction_");
+    EXPECT_THAT(multiplicity_correction_, NotNull());
```

Comment:
ASSERT_THAT here

Reply:
Well. it's in constructor, so this cant't be ASSERT_THAT. I'll move this to SetUp